tests/framework/db/schema/CMssqlCommandBuilderTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+class CMssqlCommandBuilderTest extends CTestCase
+{
+	/**
+	 * @var CDbConnection
+	 */
+	private $db;
+
+	/**
+	 * Mock a CDbConection, with a CMssqlSchema and CMssqlTableSchema
+	 */
+	public function setUp()
+	{
+		/*
+		 * Disable the constructor and mock `open` so that CDbConnection does not
+		 * try to make a connection
+		 */
+		$this->db = $this->getMockBuilder(CDbConnection::class)
```

Comment:
@wtommyw `::class` is not available in older PHP versions, see the failed CI test. Classname as string should work.